### ISSUE TITLE
SortTypes issue

### ISSUE THREAD

**Issue by NikYushchenko** [acacode/swagger-typescript-api#361]
Hello. I caught one issue with the sort function.

![image](https://user-images.githubusercontent.com/51864449/170973825-189d4309-b54a-4672-be69-b9a368934115.png)

```
content: schema.rawTypeData['$parsed'].content.sort(sortByName)}
```

After some research I've found that in openapi we have this
` "TriggerAPISchema": "title":"TriggerAPISchema","type":"object","properties":{}}`
which in the code has `content: 'object'`, where `sort` is undefined.

I think it would be good to add validation `Array.isArray(schema.rawTypeData['$parsed'].content)`
![image](https://user-images.githubusercontent.com/51864449/170977721-0e126a8f-909d-461b-abc2-3ef2d9234708.png)

https://github.com/acacode/swagger-typescript-api/pull/299

**Comment by andersfylling** [comment]
Perhaps bring this up here: https://github.com/acacode/swagger-typescript-api/issues/360

 - [@grandsilence](https://github.com/grandsilence)

**Comment by NikYushchenko** [comment]
Was fixed in [10.0.2](https://github.com/acacode/swagger-typescript-api/releases/tag/10.0.2)

**Comment by NikYushchenko** [comment]
Thanks
